lib/internal/event_target.js

```diff
@@ -86,7 +86,9 @@ class Event {
     if (arguments.length === 0)
       throw new ERR_MISSING_ARGS('type');
     if (options !== null)
-      validateObject(options, 'options');
+      validateObject(options, 'options', {
+        allowArray: true, allowFunction: true,
```

Comment:
Shouldn't you pass `nullable: true` instead of having a `if (options !== null)`?

Reply:
Right, this became possible when <https://github.com/nodejs/node/pull/35806> made `options` default to `null`.